backend/src/packages/chaiNNer_standard/image/batch_processing/split_spritesheet.py

```diff
@@ -60,13 +72,24 @@ def split_spritesheet_node(
     individual_w = w // columns
 
     def get_sprite(index: int):
-        row = index // columns
-        col = index % columns
+        if order == OrderEnum.ROW_MAJOR:
+            row = index // columns
+            col = index % columns
+
+            sprite = sprite_sheet[
+                row * individual_h : (row + 1) * individual_h,
+                col * individual_w : (col + 1) * individual_w,
+            ]
+        elif order == OrderEnum.COLUMN_MAJOR:
+            col = index // rows
+            row = index % rows
 
-        sprite = sprite_sheet[
-            row * individual_h : (row + 1) * individual_h,
-            col * individual_w : (col + 1) * individual_w,
-        ]
+            sprite = sprite_sheet[
+                row * individual_h : (row + 1) * individual_h,
+                col * individual_w : (col + 1) * individual_w,
+            ]
```

Comment:
Those `sprite = sprite_sheet[...]` are identical in both cases, so they can be moved outside the if/else.